test/Base_Tests/src/Runtime/State_Spec.enso

```diff
@@ -21,6 +23,33 @@ add_specs suite_builder = suite_builder.group "State" group_builder->
 
         Meta.type_of p . should_equal Unsupported_Argument_Types
 
+    fill_field =
+        (State.get Lazy_Data) + 1
+
+    group_builder.specify "lazy field and state: alloc and get" <|
+        fourty_two = State.run Lazy_Data 41 <|
+            data = Lazy_Data.Value fill_field
+            data.lazy_field
+
+        fourty_two . should_equal 42
+
+    group_builder.specify "lazy field and state: just alloc" <|
+        data = State.run Lazy_Data 41 <|
+            Lazy_Data.Value fill_field
+
+        fourty_two = Panic.recover Uninitialized_State
+            data.lazy_field
+
+        fourty_two . should_fail_with Uninitialized_State
```

Comment:
This pattern should be handled by `Test.expect_panic`

Reply:
Yes, 53c4923cd4 is shorted.